### PR TITLE
[strings] Translate detailed railway types

### PR DESCRIPTION
**Pull request by pastk**
Please send in translations for your language, thanks!
For the context: https://github.com/organicmaps/organicmaps/issues/2844#issuecomment-1599433597

(@biodranik I wanted to try the deepl translation script, but I can't register to get a free deepl api token)

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -19594,35 +19594,50 @@
 
   [type.railway.rail.highspeed]
     ref = type.railway.rail
+    en = High-speed Railway
```
Comment (fitojb) [inline data/strings/types_strings.txt:19597]:
```suggestion
    en = High-speed Railway
    es = Tren de alta velocidad
```

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -19594,35 +19594,50 @@
...
+    ru = Высокоскоростная железная дорога
 
   [type.railway.rail.tourism]
     ref = type.railway.rail
+    en = Touristic Railway
```
Comment (fitojb) [inline data/strings/types_strings.txt:19602]:
```suggestion
    en = Touristic Railway
    es = Tren turístico
```

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -19594,35 +19594,50 @@
...
+    ru = Туристическая железная дорога
 
   [type.railway.rail.main]
     ref = type.railway.rail
+    en = Railway
```
Comment (fitojb) [inline data/strings/types_strings.txt:19607]:
```suggestion
    en = Railway
    es = Tren
```

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -19594,35 +19594,50 @@
...
+    ru = Железная дорога
 
   [type.railway.rail.branch]
     ref = type.railway.rail
+    comment = Includes ordinary railway=rail w/o more specific usage= and service= tags.
+    en = Railway Branch
```
Comment (fitojb) [inline data/strings/types_strings.txt:19613]:
```suggestion
    en = Railway Branch
    es = Ramal ferroviario
```

File: data/strings/types_strings.txt [hunk 5]
```diff
@@ -19594,35 +19594,50 @@
...
+    ru = Периферийная железная дорога
 
   [type.railway.rail.utility]
     ref = type.railway.rail
+    comment = Non-passenger utility tracks: industrial, military, test.
+    en = Utility Railway
```
Comment (fitojb) [inline data/strings/types_strings.txt:19619]:
```suggestion
    en = Utility Railway
```
```suggestion
    en = Utility Railway
    es = Tren de servicios
```

File: data/strings/types_strings.txt [hunk 6]
```diff
@@ -19594,35 +19594,50 @@
...
+    ru = Служебная железная дорога
 
   [type.railway.rail.spur]
     ref = type.railway.rail
-    en = Spur Rail
+    en = Railway Spur
     af = Syspoor
     de = Anschlussgleis
     et = Rööbastee
```
Comment (fitojb) [inline data/strings/types_strings.txt:19627]:
```suggestion
    et = Rööbastee
    es = Espolón ferroviario
```

File: data/strings/types_strings.txt [hunk 7]
```diff
@@ -19594,35 +19594,50 @@
...
     pt-BR = Ramal de ferrovia
-    ru = Подъездная Ж/Д
+    ru = Подъездной Ж/Д путь
     tr = Düz Ray
     uk = Під'їздна залізниця
 
   [type.railway.rail.service]
     ref = type.railway.rail
+    comment = Short service tracks: siding, yard, crossover.
+    en = Service Rail Track
```
Comment (fitojb) [inline data/strings/types_strings.txt:19639]:
```suggestion
    en = Service Rail Track
    es = Vía de maniobras
```

**Review by fitojb — CHANGES_REQUESTED** [review]
Spanish suggestions below

File: data/strings/types_strings.txt (hunk 1 above)
Comment (Wivik) [inline data/strings/types_strings.txt:19597]:
```suggestion
    en = High-speed Railway
    fr = Train à grande vitesse
```

File: data/strings/types_strings.txt (hunk 2 above)
Comment (Wivik) [inline data/strings/types_strings.txt:19602]:
```suggestion
    en = Touristic Railway
    fr = Ligne touristique
```

File: data/strings/types_strings.txt (hunk 3 above)
Comment (Wivik) [inline data/strings/types_strings.txt:19607]:
```suggestion
    en = Railway
    fr = Chemin de fer
```

File: data/strings/types_strings.txt (hunk 5 above)
Comment (Wivik) [inline data/strings/types_strings.txt:19619]:
```suggestion
    en = Utility Railway
    fr = Train de fret
```

File: data/strings/types_strings.txt [hunk 8]
```diff
@@ -19594,35 +19594,50 @@
...
   [type.railway.rail.utility]
     ref = type.railway.rail
+    comment = Non-passenger utility tracks: industrial, military, test.
+    en = Utility Railway
+    ru = Служебная железная дорога
 
   [type.railway.rail.spur]
     ref = type.railway.rail
-    en = Spur Rail
+    en = Railway Spur
```
Comment (Wivik) [inline data/strings/types_strings.txt:19624]:
```suggestion
    en = Railway Spur
    fr = Embranchement ferroviaire
```

File: data/strings/types_strings.txt (hunk 7 above)
Comment (Wivik) [inline data/strings/types_strings.txt:19639]:
```suggestion
    en = Service Rail Track
    fr = Service ferroviaire
```

**Review by Wivik — COMMENTED** [review]
Here is the French lines.

File: data/strings/types_strings.txt (hunk 8 above)
Comment (LaoshuBaby) [inline data/strings/types_strings.txt:19624]:
```suggestion
    en = Railway Spur
    zh-Hans = 铁路岔线
```

File: data/strings/types_strings.txt (hunk 7 above)
Comment (LaoshuBaby) [inline data/strings/types_strings.txt:19639]:
```suggestion
    en = Service Rail Track
    zh-Hans = 服务性铁路
```

**Review by LaoshuBaby — COMMENTED** [review]
Here are Chinese (Simplified) translation

File: data/strings/types_strings.txt (hunk 1 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19597]: en = High-speed Railway
pt = Ferrovia de alta velocidade

File: data/strings/types_strings.txt (hunk 2 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19602]: en = Touristic Railway
pt = Ferrovia turística

File: data/strings/types_strings.txt (hunk 3 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19607]: en = Railway
pt = Ferrovia

File: data/strings/types_strings.txt (hunk 4 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19613]: en = Railway Branch
pt = Ramal ferroviário

File: data/strings/types_strings.txt (hunk 5 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19619]: en = Utility Railway
pt = Ferrovia de serviços

File: data/strings/types_strings.txt [hunk 9]
```diff
@@ -19594,35 +19594,50 @@
...
     af = Syspoor
     de = Anschlussgleis
     et = Rööbastee
     ja = 引き込み線
     nl = Zijspoor
     pt = Ramal de ferrovia
     pt-BR = Ramal de ferrovia
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19631]: pt = Ramal de inserção
pt-BR = Ramal de inserção

File: data/strings/types_strings.txt (hunk 7 above)
Comment (matheusgomesms) [inline data/strings/types_strings.txt:19639]: pt = Linha de serviço

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
Added PT/PT-BR

File: data/strings/types_strings.txt (hunk 1 above)
Comment (strump) [inline data/strings/types_strings.txt:19597]: `uk = Швидкісна залізниця`

File: data/strings/types_strings.txt (hunk 2 above)
Comment (strump) [inline data/strings/types_strings.txt:19602]: `uk = Туристична залізниця`

File: data/strings/types_strings.txt (hunk 4 above)
Comment (strump) [inline data/strings/types_strings.txt:19613]: `uk = Залізнична гілка`

**Review by Supaplextw — COMMENTED** [review]
add zh-Hant translation

File: data/strings/types_strings.txt [hunk 10]
```diff
@@ -19594,35 +19594,50 @@
...
     uk = Під'їздна залізниця
 
   [type.railway.rail.service]
     ref = type.railway.rail
+    comment = Short service tracks: siding, yard, crossover.
+    en = Service Rail Track
+    ru = Вспомогательный Ж/Д путь
 
   [type.railway.rail.bridge]
     en = Railway Bridge
```
Comment (Supaplextw) [inline data/strings/types_strings.txt:19643]: zh-Hant = 鐵路橋

**Review by matheusgomesms — APPROVED** [review]

**Review by vng — APPROVED** [review]

**Pull request merged**